### PR TITLE
[feature3] Feature 3 Implementation with added 3rd file

### PR DESCRIPTION
**Pull request by jmbo1190**
New implementation of Feature 3 

- including added feat-3-file3.txt

### CODE REVIEW

File: feature-3/feat-3-file1.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This is the first file created in feature3 branch.
+
+[more info to be added...]
```
Comment (jbodart-argenx) [inline feature-3/feat-3-file1.txt:3]: This line needs to be finalised

File: feature-3/feat-3-file2.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+# This is the second file created in feature3 branch
```
Comment (jbodart-argenx) [inline feature-3/feat-3-file2.txt:1]: please add another line here

**Review by jbodart-argenx — CHANGES_REQUESTED** [review]
please implement requested changes

File: feature-3/feat-3-file1.txt (hunk 1 above)
Comment (jmbo1190) [reply feature-3/feat-3-file1.txt]: done

File: feature-3/feat-3-file2.txt (hunk 2 above)
Comment (jmbo1190) [reply feature-3/feat-3-file2.txt:1]: other line added

**Comment by jbodart-argenx** [comment]
Good

**Review by jbodart-argenx — APPROVED** [review]

**Pull request merged**